cmd/credentialUtil.go

```diff
@@ -681,7 +673,20 @@ func createBlobPipeline(ctx context.Context, credInfo common.CredentialInfo, log
 		nil,
 		ste.NewAzcopyHTTPClient(frontEndMaxIdleConnectionsPerHost),
 		nil, // we don't gather network stats on the credential pipeline
-	), nil
+	)
+}
+
+func createBlobPipeline(ctx context.Context, credInfo common.CredentialInfo, logLevel pipeline.LogLevel) (pipeline.Pipeline, error) {
```

Comment:
Why did we separate these cases?

Reply:
Technically, we could just add proper support for shared key to the blob endpoint and not hack around it. But I don't think we want to encourage customers to use shared key in the first place; so this was to avoid adding support for that in directly.